addons/wowl/static/src/services/rpc.ts

```diff
@@ -65,6 +67,7 @@ function jsonrpc(env: OdooEnv, url: string, params: Params): Promise<any> {
         type: "network",
       };
       bus.trigger("RPC_ERROR", error);
+      bus.trigger("RPC:RESPONSE", data.id);
```

Comment:
I think triggering twice on a bus is not really nice to see. But I guess this is acceptable here.

Reply:
I'm not a big fan too but I don't found a better solution.